hooks/scripts/validate-author-email.sh

```diff
@@ -1,11 +1,8 @@
 #!/bin/bash
 
-AUTHOR_EMAIL=`git config user.email`
-DOMAIN=$1
-
-if [[ $DOMAIN = "" ]]; then
-  DOMAIN="@flux7.com"
-fi
+AUTHORINFO=$(git var GIT_AUTHOR_IDENT) || exit 1
+AUTHOR_EMAIL=$(printf '%s\n\n' "${AUTHORINFO}" | sed -n 's/^.* <\(.*\)> .*$/\1/p')
```

Comment:
@mukesh-flux7 I think now that the issue with the script arguments is fixed, using the `git config user.email` will work now. 

`git var` was introduced in git 2.18 which is fairly recent but this check might be kinda flakey.
```suggestion
AUTHOR_EMAIL=$(git config user.mail)
```